spec/api.spec.js

```diff
@@ -268,35 +271,33 @@ describe('API', function () {
 
       const err = new Error('tokens are missing')
       err.tokens = tokenList
-      hMock = sinon.mock(hikaru)
 
       onCluster = (opts, cluster) => {
         opts.ip = cluster.masterEndpoint
       }
 
-      const firstArgs = Object.assign(
-        {},
-        hikaruSpec,
-        {
-          data: { cluster: clusterDetail },
-          onCluster: onCluster
+      const firstArgs = {
+        ...hikaruSpec,
+        data: {
+          cluster: { ...clusterDetail }
+        }
+      }
+      const secondArgs = {
+        ...hikaruSpec,
+        data: {
+          ...specData,
+          cluster: { ...clusterDetail }
         },
-        { onCluster }
-      )
-      firstArgs.data.cluster = clusterDetail
-      console.log('first', firstArgs)
-      hMock.expects('deployCluster')
-        .withArgs(specification, firstArgs)
-        .once()
-        .rejects(err)
+        onCluster
+      }
 
-      const secondArgs = Object.assign({}, hikaruSpec, { data: specData }, { onCluster })
-      secondArgs.data.cluster = clusterDetail
-      console.log('second', secondArgs)
-      hMock.expects('deployCluster')
-        .withArgs(specification, secondArgs)
-        .once()
+      hikaru.deployCluster = sinon.stub()
```

Comment:
sinon mocks dont support multiple calls on mocked methods, so I had to change this to a stub.